src/component/ImageUploader.tsx

```
@@ -0,0 +1,58 @@
+      reader.readAsDataURL(file);
+    }
+  };
+
+  return (
+    <div className="inline-flex flex-col items-start justify-center gap-2">
+      <label htmlFor="image-uploader">Label</label>
+      <div className="flex">
+        <div
+          className="border-primary-0 relative mr-3 flex h-32 w-32 items-center justify-center rounded-md border-2 border-dashed"
+          onClick={handleClick}
+        >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)